forte/processors/allennlp_processors.py

```diff
@@ -48,8 +51,13 @@ def initialize(self, resources: Resources, configs: Config):
 
         if configs.tag_formalism not in MODEL2URL:
             raise ProcessorConfigError('Incorrect value for tag_formalism')
-        model_url = MODEL2URL[configs.tag_formalism]
-        self.predictor: Predictor = Predictor.from_path(model_url)
+        if configs.tag_formalism == 'stanford':
+            self.predictor = {
+                'stanford': Predictor.from_path(MODEL2URL['stanford'])}
+        if configs.tag_formalism == 'srl':
```

Comment:
Please read the documentation for the meaning of `config` here: https://github.com/asyml/forte/blob/3a09d8d7db4a79e0b7f65aebf8ae0b1902dd1d91/forte/processors/allennlp_processors.py#L93

`tag_formalism` is only applicable to the `parsing` predictor. What you should check here is `processors` contains `srl`

Reply:
Got it. Will modify it